content/classes/exercises/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Exercises"
+date: 2023-01-23T13:36:52-06:00
+draft: false
+weight: 2
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Before you get started, make sure you have forked and cloned the [starter code](https://github.com/LaunchCodeEducation/csharp-web-dev-exercises) repository for the exercises.
```

Comment:
I'm getting this error when I run the starter code with the below solution -> Program does not contain a static 'Main' method suitable for an entry point

Reply:
Will add a link back to place where we discuss Main() since this is an expected error.